kadena-beta/docs/Kadena-README.md

```diff
@@ -1,10 +1,40 @@
-# Kadena Blockchain Documentation
+# Kuro Blockchain Documentation
 ---
 
-Kadena Version: 1.1.x
+Kuro Version: 1.1.x
```

Comment:
I think we're well past this version, maybe just delete this line

Reply:
done